kustomize/base/all-in-one/resources/deploy-config.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
@fjtirado I'm pretty sure @r00ta removed these files as part of his work and we don't need them back. @r00ta can you confirm I'm correct?

Reply:
yes, the eclipse merge tool trick me